guides/common/modules/ref_virt-who-configuration-examples.adoc

```diff
@@ -0,0 +1,98 @@
+[id="virt-who-configuration-examples_{context}"]
+= Virt-who configuration examples
+
+You configure virt-who in a file that specifies details such as the virtualization type and the hypervisor or virtualization manager to query.
+
+[IMPORTANT]
+====
+The `rhevm` and `xen` hypervisors are not supported.
+
+The `kubevirt` hypervisor is a Technology Preview.
+====
+
+.Generic example
+
+[source,text]
```

Comment:
```suggestion
[options="nowrap" subs="+quotes,verbatim,attributes"]
```
Attributes have to be enabled for the replacement to work.

Reply:
Right! Forgot about that.